docs/BUILD.md

```diff
@@ -627,3 +629,16 @@ Last but not least, is this completely necessary?
 
 * `images/*.yml`: The need to update `images/*.yml` is understood, as these packages change a lot. Even so, the utilities might be better structured as a separate external utility that is run manually. Most of the time, you just build from a static `rootfs.yml` or `installer.yml`. When you want to update them, you run `update-images` (or similar) and it updates them. Finally, you commit when a good build is ready.
 * `pkg/*/Dockerfile`: The need to generate `Dockerfile` for many of the packages may mean too much cross-dependency, which can be brittle. It is possible that this is necessary, and there is no other way around it. However, we should explore how we can simplify dependencies.
+
+## How to update eve-alpine package
+
+Unlike a lot of other Linux distributions, Alpine Linux doesn't provide historical versions of all its packages. In fact, Alpine Linux reserves the right to update packages with security patches behind the scenes resulting in content of Alpine x.y.z repositories shifting slightly from time to time. This, obviously, goes against the principle of reproducible builds and makes our `lfedge/eve-alpine` cache serve a double function: not only it is used to speed up the build, but it also may end up being the only place on the Internet where a certain Alpine Linux package version x.y.z could be available from.
+
+The later aspect makes maintaining `lfedge/eve-alpine` a bit tricky, even though at its core it is simply driven by the list of packages recorded in the manifest files under [pkg/alpine/mirrors](../pkg/alpine/mirrors). Removing packages from the cache is not advisable (and should really only be done during major EVE version updates). Adding packages to the cache consists of two steps:
+
+1. adding new package names to the manifest files
+2. updating `FROM ... AS cache` line in [pkg/alpine/Dockerfile](../pkg/alpine/Dockerfile) to point to the last version of the cache
```

Comment:
There is no such line in the Dockerfile. Did you mean [this line](https://github.com/lf-edge/eve/blob/master/pkg/alpine/Dockerfile#L4) with `as mirror`?

> last version of the cache

This is a bit vague. If all of this means, the base version of alpine on which it is built, then let's be explicit about it.

What happens then when we bump to, say, alpine 3.14?

Reply:
> There is no such line in the Dockerfile. 

There is -- it just happens to be currently commented out

> This is a bit vague. If all of this means, the base version of alpine on which it is built, then let's be explicit about it.

No, it literally means the latest version of the `lfedge/eve-alpine` package itself -- not sure how to described it better (but I think example is doing a good job of explaining it)